Scripts/World/world_manager.gd

```diff
@@ -24,10 +24,19 @@ var current_player : Player
 var selected_hero : ArmyForm
 var combat_tile : Vector2i
 
+var _batch_mode : bool = false
+
 #endregion
 
 signal world_move_done
 
+
+func _process(_delta):
+	if Input.is_action_just_pressed("PRINT_WORLD_STATE"):
```

Comment:
as earlier, i'd keep all Input.is_action_just_pressed in input manager
so there is a single place to check for key bindings tweaks
not whole code base

Reply:
with IM.quit_game() doubly concerning